src/conn/conn_dhandle.c

```diff
@@ -125,18 +125,13 @@ __wt_conn_dhandle_find(
 static int
 __conn_dhandle_mark_dead(WT_SESSION_IMPL *session)
```

Comment:
We don't need a separate function just to set a flag -- I guess I should merge it?